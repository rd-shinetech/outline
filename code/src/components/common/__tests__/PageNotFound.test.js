import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import PageNotFound from '../PageNotFound';


let props = undefined;


beforeAll(() => {
    props = { location:{pathName: '/blah'}};
    return props;
});



describe('PageNotFound.test.js', () => {
    let wrapper = undefined;

    beforeEach(() => {
        wrapper = shallow(<PageNotFound {...props}/>);
        return wrapper;
    });

    it('renders without crashing', () => {
        expect(wrapper.length).toEqual(1);
    });


    it('renders as expected', () => {
        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    });


});
