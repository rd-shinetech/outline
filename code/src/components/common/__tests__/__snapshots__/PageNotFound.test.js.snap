exports[`PageNotFound.test.js renders as expected 1`] = `
<div
  className="jumbotron">
  <h1
    className="display-1">
    Page Not Found
  </h1>
  <p
    className="lead">
    404 Error
  </p>
  <p>
    No match for the link 
    <code />
  </p>
</div>
`;
